alerter_test.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <assert.h>
+#include "alerter.h"
+
+int main() {
+    
+    assert(alertFailureCount == 0);
```

Comment:
Make sure you have covered at least 3 values i.e. before and after threshold value and threshold value.
This makes the expectation from production code clear.